src/components/Sidebar/components/SidebarItem/index.tsx

```diff
@@ -0,0 +1,37 @@
+type SidebarItemProps = {
+  title: string;
+  workspaceProject: string;
+  activeContent: string;
+  setActiveContent: (content: string) => void;
+};
+
+export const SidebarItem = ({
+  title,
+  workspaceProject,
+  activeContent,
+  setActiveContent,
+}: SidebarItemProps) => {
+  const handleClick = () => {
+    setActiveContent(`${workspaceProject}-${title}`);
+  };
+
+  const handleKeyDown = (event: React.KeyboardEvent) => {
+    if (event.key === 'Enter' || event.key === ' ') {
+      setActiveContent(`${workspaceProject}-${title}`);
+    }
+  };
+
+  const isActive = activeContent === `${workspaceProject}-${title}`;
+
+  return (
+    <div
+      className={`sidebar__item ${isActive ? 'sidebar__item--active' : ''}`}
+      role="button"
+      tabIndex={0}
```

Comment:
Your treating a `div` as a `button` and that is generally a bad idea. I think your doing it so as to avoid using `ul` and `li` elements since bootstrap is adding extra CSS to those. If that is the case, doing it this way I don't think is the right approach. It is easy enough in your CSS to override what bootstrap is doing, which as far as I can tell is mandating the style and a bit of padding, so do use `ul` and `li`, but on the `ul` set:

```css
list-style-type: none;
padding-left: unset;
```
It is possible that we can even do this in the main SCSS file

```scss
li {
  list-style-type: none;
  padding-left: unset;
}
```

this seems better to me, than abusing the div, which will just have to be redone if/when we fix the bootstrap bleed-through. This is a styling issue caused by bootstrap in the web presence, not a semantic HTML issue and it is in the styling that the fix should be applied.

I have started adding 

```html
    <link
      rel="stylesheet"
      type="text/css"
      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css"
    />
```
into the head of the index.html of the resource catalogue UI, so I can see what bad things bootstrap is doing and work around it. Of course that change should never be committed....
I also think we might be able to solve this bleed-through of bootstrap styling by adding something like [normalize.css](https://www.npmjs.com/package/normalize.css/v/3.0.3). Right now that is just an idea though, I've yet to prove that will solve the problem.

Reply:
Great. I have kept this as a reference in my notes, I will be rewriting all of the UI components in a later PR, so I will make sure to follow this convention.